src/main/java/redis/clients/jedis/JedisPubSub.java

```diff
@@ -15,11 +15,9 @@
 import redis.clients.jedis.exceptions.JedisException;
 import redis.clients.util.SafeEncoder;
 
-public abstract class JedisPubSub {
+public abstract class JedisPubSub extends BinaryJedisPubSub {
```

Comment:
K, I believe the "proper" way to proceed would be to subclass and downmerge without subclassing.. u ok with this @HeartSaVioR ?